data/35e/wizards_of_the_coast/campaign_settings/forgotten_realms/monsters_of_faerun/mon_races.lst

```diff
@@ -4,7 +4,7 @@ SOURCELONG:Monster Compendium: Monsters of Faerun	SOURCESHORT:mon	SOURCEDATE:200
 ###Block: Base
 #Race Name													Favored Class		Starting Feats	Size		Move			Face/Space	Reach		Vision						Bonus Languages							Stat bonus											Racial HD Class Skills					Automatically Gained Language		Ability												Nb Legs	Nb Hands	Natural Attacks											Monster Class Name and Starting Level	Main Race Type	Race Subtype								Type		Level Adjustment	Challenge Rating	Source Page
 # Race Name		Unique Key						Favored Class		Starting Feats	Size		Move			Face/Space	Reach		Vision						Bonus Languages							Stat bonus											Racial HD Class Skills					Auto Language				Ability												Nb Legs	Nb Hands	Natural Attacks											Monster Class Name and Starting Level	Main Race Type	Race Subtype								Type		Level Adjustment	Challenge Rating	Source Page		SORTKEY
-Artic Dwarf		KEY:Dwarf (Artic)					FAVCLASS:Fighter|Ranger	STARTFEATS:1	FACT:HasFirstLevelFeat|True	SIZE:S	MOVE:Walk,20	FACE:5	REACH:5	VISION:Darkvision (60')				LANGBONUS:Giant,Gnome,Goblin,Orc,Terran,Undercommon	BONUS:STAT|STR|4	BONUS:STAT|DEX|-2	BONUS:STAT|CON|2	BONUS:STAT|CHA|-2									AUTO:LANG|Common|Dwarven		ABILITY:Internal|AUTOMATIC|Dwarf Artic Racial Trait|Racial Traits ~ Dwarf	LEGS:2	HANDS:2																					RACETYPE:Humanoid	RACESUBTYPE:Dwarf								TYPE:PC	LEVELADJUSTMENT:2	CR:1/2		SOURCEPAGE:p.	SORTKEY:Dwarf
+Artic Dwarf		KEY:Dwarf (Artic)					FAVCLASS:Fighter|Ranger	STARTFEATS:1	FACT:HasFirstLevelFeat|True	SIZE:S	MOVE:Walk,20	FACE:5	REACH:5	VISION:Darkvision (60')				LANGBONUS:Giant,Gnome,Goblin,Orc,Terran,Undercommon	BONUS:STAT|STR|4	BONUS:STAT|DEX|-2	BONUS:STAT|CON|2	BONUS:STAT|CHA|-2									AUTO:LANG|Common|Dwarven		ABILITY:Internal|AUTOMATIC|Dwarf Arctic Racial Trait|Racial Traits ~ Dwarf	LEGS:2	HANDS:2																					RACETYPE:Humanoid	RACESUBTYPE:Dwarf								TYPE:PC	LEVELADJUSTMENT:2	CR:1/2		SOURCEPAGE:p.	SORTKEY:Dwarf
```

Comment:
Redundant, and not following Race standard convention. Subraces are chosen by ability.